HHS/Head-Start-TTADP
# [Prod] Properly track eventSubmitted, updates to Slack message around PR deploys

## Description of change

We need to capture the "eventSubmitted" variable in the data of a Training Report when the owner submits it. The "complete event" functionality depends on it. It seems to have been removed as part of a bug squash in the days leading up to the TR release.

This PR also includes code to further customize the slack message deployments.

## How to test
Impersonating the user in [this slack thread](https://adhoc.slack.com/archives/C022R5301L4/p1725032185745969), confirm that you can submit and then complete the event in question

Alternately, confirm "eventSubmitted" is present in the request body when submitting a training report. A unit test has been added to validate this specific field.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3336


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
